llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -543,12 +543,17 @@ defset list<VTypeInfoToWide> AllWidenableBFloatToFloatVectors = {
 // This represents the information we need in codegen for each pseudo.
 // The definition should be consistent with `struct PseudoInfo` in
 // RISCVInstrInfo.h.
-class RISCVVPseudo {
+class RISCVVPseudo<dag outs, dag ins, list<dag> pattern, string opcodestr = "", string argstr = "">
```

Comment:
Most of this change seems like an NFC to have RISCVPseudo sub-class Pseudo.  That seems like a reasonable change on it's own merits and should probably be extracted separately.